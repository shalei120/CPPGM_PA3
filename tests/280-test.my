error
error
4
2687600754856047956
2
4294967295u
4294967295u
18446744073709551615u
18446744073709551615u
-8070450532247928833
8070450532247928833u
34u
35u
34u
35u
23
0
18446744073709551580u
eof
