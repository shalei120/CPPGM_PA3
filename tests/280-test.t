"fgdf"  "sdfgds""sdfg"
"dfgs""sdfgs""sdfgs"

0?21:0?34:45?5?4:3:6

0x239238282212323 + 0x2313232321231231
true + true
0xffffffff
0xffffffffu
0xffffffffffffffffu
0xffffffffffffffff
0x7fffffffffffffff + 0x1000000000000000
~0x8fffffffffffffff +1
true?34u: 35
false?34u: 35
true?34: 35u
false?34: 35u

34u?23:34

!0x23u
~0x23u  //00000000 00000000 00000000 00100011  
	//11111111 11111111 11111111 11011100
