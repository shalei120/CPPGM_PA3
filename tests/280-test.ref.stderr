ERROR: string-literal in controlling expression
ERROR: string-literal in controlling expression
ERROR: string-literal in controlling expression
ERROR: string-literal in controlling expression
ERROR: string-literal in controlling expression
ERROR: string-literal in controlling expression
